### PR TITLE
Initial draft of docs for chartering

### PR DESCRIPTION
**Pull request by mhdawson**
Initial version of docs for chartering

Signed-off-by: Michael Dawson <michael_dawson@ca.ibm.com>

### CODE REVIEW

File: Governance.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+Prior to each WG meeting the moderator will share the Agenda with
+members of the WG. WG members can add any items they like to the
+agenda at the beginning of each meeting. The moderator and the WG
+cannot veto or remove items.
+
+The moderator is responsible for summarizing the discussion of each
+agenda item and sends it as a pull request after the meeting.
+
+### Repository Management
```
Comment (mhdawson) [inline Governance.md:82]: All repositories under the management of the package maintenance team must include:

* LICENCE.md from the list approved by the OpenJS Foundation
* CODE_OF_CONDUCT.md referencing the Node.js Code of conduct in the admin repo.
* CONTRIBUTING.md which includes the current version of the DCO used by the Node.js Project
* An PR template(s) used for all Pull requests which includes the current version of the DCO used by the Node.js Project.
* A README.md which includes references to the GOVERNANCE.md in the package-maintenance repository as the authoritative governance which applies to the repository.

**Comment by mhdawson** [comment]
@nodejs/package-maintenance first cut at governance needed related to chartering the package maintenance team as a Working Group.

File: Charter.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## Working Group Charter Template
+
+The Node.js Package Maintenance Working Group is jointly governed by the members
+of that Working Group (WG).
+
+The WG responsibilities include:
+
+* Management of repositorities within the [pkgjs](https://github.com/pkgjs)
```
Comment (styfle) [inline Charter.md:8]:
```suggestion
* Management of repositories within the [pkgjs](https://github.com/pkgjs)
```

File: Charter.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  GitHub organization including but not limited to:
+    * Managing the list of organization owners which supplement the standard
+      Node.js organization owners as outlined in:
+[https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners](https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners)
+    * Approving new repositories
+    * Managing the maintainer teams for all of the repositories.
+    * Contribution policy for repositories
+* Technical direction for the projects within
+  the [pkgjs](https://github.com/pkgjs)  orgnanization
```
Comment (styfle) [inline Charter.md:17]:
```suggestion
  the [pkgjs](https://github.com/pkgjs)  organization
```

File: Governance.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+
+### Collaborators
+
+### WG Membership
+
+The package-maintence team has two levels of membership. Administrative
+members and regular members. 
+
+If you'd like to be listed as regular team member, open a PR adding yourself
+to this [MEMBERS.md](MEMBERS.md) along with a few words on how you are planning
```
Comment (styfle) [inline Governance.md:11]:
```suggestion
to [MEMBERS.md](MEMBERS.md) along with a few words on how you are planning
```

File: Governance.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+##### Maintainers
+
+Maintainers for the nodejs/package-maintenance repo are managed
+through the [package-maintenance](https://github.com/orgs/nodejs/teams/package-maintenance/)
+team.  Administrative members are given the maintainer fole for that team
+and can add/remove members as they join or leave the team.
+
+##### Landing PRs
+
+The package maintenance team policy on landing a PR in this respositry
```
Comment (styfle) [inline Governance.md:102]:
```suggestion
The package maintenance team policy on landing a PR in this repository
```

File: Governance.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+##### Maintainers
+
+Maintainers for this repository are managed
+through the [ci-condifg-travis-maintainers](https://github.com/orgs/nodejs/teams/ci-config-travis-maintainers/)
+team. Administrative members are given the maintainer fole for that team
+and can add/remove members as appropriate.
+
+##### Landing PRs
+
+The package maintenance team policy on landing a PR in this respositry
```
Comment (styfle) [inline Governance.md:121]:
```suggestion
The package maintenance team policy on landing a PR in this repository
```

File: README.md [hunk 7]
```diff
@@ -65,15 +80,17 @@ to be accepted: it is a manual workflow, so it could take some days (we are work
 
 ## Pull Request Merging Policy
 
-The package maintenance team policy on PR closure is for there to be
-- At least 4 approvals
+The package maintenance team policy on landing a PR n the nodejs/package-maintenance
+repository is for there to be:
+- At least 4 approvals from rebular members
```
Comment (styfle) [inline README.md:85]:
```suggestion
- At least 4 approvals from regular members
```

I'm not sure what the intended word here is but I think you mean "regular"?

File: Charter.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
+## Working Group Charter Template
```
Comment (wesleytodd) [inline Charter.md:1]:
```suggestion
## Package Maintenance Working Group Charter
```

File: Charter.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+of that Working Group (WG).
+
+The WG responsibilities include:
+
+* Management of repositorities within the [pkgjs](https://github.com/pkgjs)
+  GitHub organization including but not limited to:
+    * Managing the list of organization owners which supplement the standard
+      Node.js organization owners as outlined in:
+[https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners](https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners)
+    * Approving new repositories
```
Comment (wesleytodd) [inline Charter.md:13]: Assuming this follows the same policy we have of blocking via dissent, otherwise moving forward, would it not be wise to rephrase this to something more like `Overseeing repositories (creating, moving, removing)`?  I feel like the goal is to keep the barriers low to keep productivity high, this wording feels heavy handed to me.

File: Governance.md [hunk 10]
```diff
@@ -0,0 +1,234 @@
+## Template: Package Maintenance Working Group Governance
```
Comment (wesleytodd) [inline Governance.md:1]:
```suggestion
## Package Maintenance Working Group Governance
```

File: Governance.md [hunk 11]
```diff
@@ -0,0 +1,234 @@
...
+
+In certain cases, OpenJS Cross Project Council and/or OpenJS Foundation Board
+of Directors approval may also be required. (Most likely in the case of
+transfeering a repository into or out of the organization).
+
+##### Maintainers
+
+Maintainers for the repositories in the Pkgjs repository are managed
+through the [pkgjs-maintainers](https://github.com/orgs/nodejs/teams/pkgjs-maintainers/)
+team.  Administrative members are given the maintainer fole for that team
```
Comment (wesleytodd) [inline Governance.md:175]:
```suggestion
team.  Administrative members are given the maintainer role for that team
```

File: Governance.md [hunk 12]
```diff
@@ -0,0 +1,234 @@
...
+and can add/remove members as appropriate.
+
+##### Landing PRs
+
+The package maintenance team policy on landing a PR in the repositories within the Pkgjs
+repository is for there to be:
+- At least 2 approvals from package-maintenance members
```
Comment (wesleytodd) [inline Governance.md:182]: I think we should change this to be 1 approvals from member. The problem right now would be we don't have 2 members watching each of the repos 😀!  I also think that we need to add a `fast-track` policy on here with the same guidelines of 1 approval, where it can circumvent the 72hour hold.  Again, with the current small contributor base, it would really slow down the effort to wait 72 hours for each merge.

**Comment by wesleytodd** [comment]
Oh, it also occurs to me that we need to align the language to be either `Working Group` or `Team`.  It uses both in the doc.

File: Charter.md [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+    * Managing the maintainer teams for all of the repositories.
+    * Contribution policy for repositories
+* Technical direction for the projects within
+  the [pkgjs](https://github.com/pkgjs)  orgnanization
+* Project governance and process (including this policy)
+* Managing the maintainer teams and contribution policies for the
+  following repositories
```
Comment (dominykas) [inline Charter.md:20]: Should we add a label and rephrase this as "Repositories with `pkgjs` label under `nodejs` organization(s)"? Not sure if we need to mention the `pkgjs` org here again.

File: Charter.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+  * nodejs/ci-config-travis
+  * nodejs/ci-config-github-actions
+  * nodejs/package-maintenance repository.
+
+For the current list of WG members, see the project [README.md][].
+
+[README.md]: ./README.md#current-project-team-members
```
Comment (dominykas) [inline Charter.md:27]: This does not display in HTML - was it meant to be included in the above line ("For the current list...")?

File: Governance.md [hunk 15]
```diff
@@ -0,0 +1,234 @@
...
+
+Changes to Administrative WG membership should be posted in
+the agenda, and may be suggested as any other agenda
+item (see "WG Meetings" below).
+
+If an addition or removal is proposed during a meeting, and the full
+WG is not in attendance to participate, then the addition or removal
+is added to the agenda for the subsequent meeting.  This is to ensure
+that all Administrative  members are given the opportunity to
+participate in all membership decisions.
```
Comment (dominykas) [inline Governance.md:43]: Don't we get "full attendance" a bit too rarely for this? Can we not leave this to be discussed in the issues or otherwise?

File: Governance.md [hunk 16]
```diff
@@ -0,0 +1,234 @@
...
+remedied by the resignation or removal of one or more
+Administrative WG members affiliated with the
+over-represented employer(s).
+
+For the current list of members, see the project [README.md][].
+
+### WG Meetings
+
+The WG meets regularly as scheduled on the Node.js
+[calendar](nodejs.org/calendar). Each meeting should be
```
Comment (dominykas) [inline Governance.md:59]:
```suggestion
[calendar](https://nodejs.org/calendar). Each meeting should be
```

File: Governance.md [hunk 17]
```diff
@@ -0,0 +1,234 @@
...
+
+The moderator is responsible for summarizing the discussion of each
+agenda item and sends it as a pull request after the meeting.
+
+### Repository Management
+All repositories under the management of the package maintenance team must include:
+
+* LICENCE.md from the list approved by the OpenJS Foundation
+* CODE_OF_CONDUCT.md referencing the Node.js Code of conduct in the admin repo.
+* CONTRIBUTING.md which includes the current version of the DCO used by the Node.js Project
```
Comment (dominykas) [inline Governance.md:87]: It would be useful to define what "DCO" is.

File: Governance.md [hunk 18]
```diff
@@ -0,0 +1,234 @@
...
+- 7 day period from the 4th approval to merging
+
+All PRs shall follow the [contributions policy](CONTRIBUTING.md)
+
+#### nodejs/ci-config-travis
+
+##### Maintainers
+
+Maintainers for this repository are managed
+through the [ci-condifg-travis-maintainers](https://github.com/orgs/nodejs/teams/ci-config-travis-maintainers/)
```
Comment (dominykas) [inline Governance.md:115]:
```suggestion
through the [ci-config-travis-maintainers](https://github.com/orgs/nodejs/teams/ci-config-travis-maintainers/)
```

File: Governance.md [hunk 19]
```diff
@@ -0,0 +1,234 @@
...
+- At least 2 approvals from package-maintenance members 
+- No blocking reviews
+- 72 hours since the PR was openened 
+
+#### nodejs/ci-config-github-actions
+
+##### Maintainers
+
+Maintainers for the repositories in the Pkgjs repository are managed
+through the [ci-condifg-github-actions-maintainers](https://github.com/orgs/nodejs/teams/ci-config-github-actions-maintainers/)
```
Comment (dominykas) [inline Governance.md:132]:
```suggestion
through the [ci-config-github-actions-maintainers](https://github.com/orgs/nodejs/teams/ci-config-github-actions-maintainers/)
```

File: Governance.md [hunk 20]
```diff
@@ -0,0 +1,234 @@
...
+is for there to be:
+- At least 2 approvals from package-maintenance members 
+- No blocking reviews
+- 72 hours since the PR was openened 
+
+#### nodejs/ci-config-github-actions
+
+##### Maintainers
+
+Maintainers for the repositories in the Pkgjs repository are managed
```
Comment (dominykas) [inline Governance.md:131]:
```suggestion
Maintainers for this repository are managed
```

File: Governance.md [hunk 21]
```diff
@@ -0,0 +1,234 @@
...
+of the Administrative members.
+
+In certain cases, OpenJS Cross Project Council and/or OpenJS Foundation Board
+of Directors approval may also be required. (Most likely in the case of
+transfeering a repository into or out of the organization).
+
+##### Maintainers
+
+Maintainers for the repositories in the Pkgjs repository are managed
+through the [pkgjs-maintainers](https://github.com/orgs/nodejs/teams/pkgjs-maintainers/)
```
Comment (dominykas) [inline Governance.md:174]: So that means every maintainer has access to every repo under pkgjs? (not opposed, just looking to clarify)

We probably also need to define who has access to publish to npm.

Do we need to mention 2FA anywhere?

File: Governance.md (hunk 12 above)
Comment (dominykas) [reply Governance.md:182]: I personally don't mind the wait period (I've been deliberately keeping my PRs open, even though I'm almost certian nobody will look), although we do need to be able to respond quickly in case of security issues or smth.

Also just to clarify - 1 approval _other_ than the person making the last commit?

File: Governance.md [hunk 22]
```diff
@@ -0,0 +1,234 @@
...
+* An PR template(s) used for all Pull requests which includes the current version of the DCO used by the Node.js Project.
+* A README.md which includes references to the GOVERNANCE.md in the package-maintenance repository as the authoritative governance which applies to the repository. 
+
+#### nodejs/package-maintenance 
+
+##### Maintainers
+
+Maintainers for the nodejs/package-maintenance repo are managed
+through the [package-maintenance](https://github.com/orgs/nodejs/teams/package-maintenance/)
+team.  Administrative members are given the maintainer fole for that team
```
Comment (dominykas) [inline Governance.md:97]:
```suggestion
team.  Administrative members are given the maintainer role for that team
```

File: Governance.md [hunk 23]
```diff
@@ -0,0 +1,234 @@
...
+
+All PRs shall follow the [contributions policy](CONTRIBUTING.md)
+
+#### nodejs/ci-config-travis
+
+##### Maintainers
+
+Maintainers for this repository are managed
+through the [ci-condifg-travis-maintainers](https://github.com/orgs/nodejs/teams/ci-config-travis-maintainers/)
+team. Administrative members are given the maintainer fole for that team
```
Comment (dominykas) [inline Governance.md:116]:
```suggestion
team. Administrative members are given the maintainer role for that team
```

File: Governance.md [hunk 24]
```diff
@@ -0,0 +1,234 @@
...
+- No blocking reviews
+- 72 hours since the PR was openened 
+
+#### nodejs/ci-config-github-actions
+
+##### Maintainers
+
+Maintainers for the repositories in the Pkgjs repository are managed
+through the [ci-condifg-github-actions-maintainers](https://github.com/orgs/nodejs/teams/ci-config-github-actions-maintainers/)
+team. Administrative members are given the maintainer fole for that team
```
Comment (dominykas) [inline Governance.md:133]:
```suggestion
team. Administrative members are given the maintainer role for that team
```

File: Governance.md (hunk 12 above)
Comment (wesleytodd) [reply Governance.md:182]: > I personally don't mind the wait period

I guess I don't really either, just wont want to accidentally add a process which blocks progress.  Are you good with the `fast-track` addition?

> Also just to clarify - 1 approval other than the person making the last commit?

Yes! 😀Should we clarify the language?

File: Governance.md (hunk 21 above)
Comment (wesleytodd) [reply Governance.md:174]: > So that means every maintainer has access to every repo under pkgjs?

I think we should create teams for each repo.  In the long run if we find this as too much work to maintain, then we can change, but I think in the short term it will be helpful to have fine grained control.  Leading into the next topic:

> We probably also need to define who has access to publish to npm.

Also, I agree on the publish to npm thing, and was thinking about using a GH action to sync lists (we would use this for Express I think).  I can look for an existing one or write one if none is available.

> Do we need to mention 2FA anywhere?

Currently I have the setting on the npm org to require it.  Is that an available setting in GitHub?  If not I think we do need to mention and require it.

File: Governance.md (hunk 15 above)
Comment (wesleytodd) [reply Governance.md:43]: I imagine we could get "full attendance" if there was clearly a voting event, right?  I am not opposed to a change, but I think there would be a difference between a regular meeting and a vote taking place.

**Comment by wesleytodd** [comment]
I created an "Admins" team in the `pkgjs` org in anticipation of this landing.  It doesn't specify, but do we have a list of initial admins?  Will we need to individually propose and approve each person? Or can we just set the current list of active members as the initial admin set?  I will also look to see if there is an api to sync the admins team to npm's org level ownership.  

For the rest of the repos, I would like to get some publisher automation setup.  In the comments @dominykas mentioned this, but would anyone be opposed to me creating teams within `pkgjs` for each repo like `nv-publishers`?  Then syncing that with npm?

File: Governance.md (hunk 21 above)
Comment (ljharb) [reply Governance.md:174]: yes, a github org can be set to require 2FA for membership (and all should be set to that)

File: Governance.md (hunk 12 above)
Comment (dominykas) [reply Governance.md:182]: Yes, having a fast-track for things like security issues or critical bugs does make sense.

File: Charter.md (hunk 9 above)
Comment (mhdawson) [reply Charter.md:13]: Works for me

File: Charter.md [hunk 25]
```diff
@@ -0,0 +1,27 @@
...
+of that Working Group (WG).
+
+The WG responsibilities include:
+
+* Management of repositories within the [pkgjs](https://github.com/pkgjs)
+  GitHub organization including but not limited to:
+    * Managing the list of organization owners which supplement the standard
+      Node.js organization owners as outlined in:
+[https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners](https://github.com/nodejs/admin/blob/master/GITHUB_ORG_MANAGEMENT_POLICY.md#owners)
+    * Approving new repositories
```
Comment (mhdawson) [inline Charter.md:13]:
```suggestion
    * Overseeing new repositories (creating, moving, removing)
```

File: Charter.md (hunk 14 above)
Comment (mhdawson) [reply Charter.md:27]: [README.md][] 
is a reference that gets satisfied by this line instead of including it directly. If you view as markdown and click the link it does the right thing.

File: Governance.md [hunk 26]
```diff
@@ -0,0 +1,237 @@
...
+
+The moderator is responsible for summarizing the discussion of each
+agenda item and sends it as a pull request after the meeting.
+
+### Repository Management
+All repositories under the management of the package maintenance team must include:
+
+* LICENCE.md from the list approved by the OpenJS Foundation
+* CODE_OF_CONDUCT.md referencing the Node.js Code of conduct in the admin repo.
+* CONTRIBUTING.md which includes the current version of the DCO used by the Node.js Project
```
Comment (mhdawson) [inline Governance.md:90]:
```suggestion
* CONTRIBUTING.md which includes the current version of the Developer Certificate of Origin (DCO) used by the Node.js Project
```

File: Governance.md (hunk 21 above)
Comment (mhdawson) [reply Governance.md:177]: @wesleytodd can you write up the text for how you think we should manage?

I agree on the 2FA side.

File: Governance.md [hunk 27]
```diff
@@ -0,0 +1,237 @@
...
+##### Landing PRs
+
+The package maintenance team policy on landing a PR in this respositry
+is for there to be:
+- At least 2 approvals from package-maintenance members 
+- No blocking reviews
+- 72 hours since the PR was openened 
+
+#### pkgjs organization
+
```
Comment (mhdawson) [inline Governance.md:148]:
```suggestion
All members of the pkgjs organization will be required to have 2FA enabled, 
and the repository will be configured to enforce this requirement.

```

File: Governance.md (hunk 21 above)
Comment (mhdawson) [reply Governance.md:179]: updated above to specify that 2FA will be a requirement for all members.

File: Governance.md [hunk 28]
```diff
@@ -0,0 +1,239 @@
...
+and can add/remove members as appropriate.
+
+##### Landing PRs
+
+The package maintenance team policy on landing a PR in the repositories within the Pkgjs
+repository is for there to be:
+- At least 2 approvals from package-maintenance members 
+- No blocking reviews
+- 72 hours since the PR was openened 
+
```
Comment (mhdawson) [inline Governance.md:190]:
```suggestion

Certain types of pull requests can be fast-tracked and may land after a shorter
delay. For example:

* Focused changes that affect only documentation and/or the test suite:
  * `code-and-learn` tasks often fall into this category.
  * `good-first-issue` pull requests may also be suitable.
* Changes that fix regressions:
  * Regressions that break the workflow (red CI or broken compilation).
  * Regressions that happen right before a release, or reported soon after.

To propose fast-tracking a pull request, apply the `fast-track` label. Then add
a comment that Collaborators may upvote.

If someone disagrees with the fast-tracking request, remove the label. Do not
fast-track the pull request in that case.

The pull request may be fast-tracked if two Collaborators approve the
fast-tracking request. To land, the pull request itself still needs two
Collaborator approvals and a passing CI.

Collaborators may request fast-tracking of pull requests they did not author.
In that case only, the request itself is also one fast-track approval. Upvote
the comment anyway to avoid any doubt.
```

File: Governance.md (hunk 12 above)
Comment (mhdawson) [reply Governance.md:187]: Added fast tracking language used in the node core repo.

Comment (mhdawson) [reply Governance.md:187]: I'd be ok with 1 approval as well, just started with what's in the Node docs.   Do we have concensus that it should be 1?

File: Governance.md (hunk 15 above)
Comment (mhdawson) [reply Governance.md]: The intent was that it be on the agenda as FYI, not that we'd need all members to be there.  The additional or removal should be based on "concensus" which does not require a vote unless there are objections, we just need to ensure that people have enough time/likelyhood of being able to make an objection.

File: Governance.md (hunk 21 above)
Comment (wesleytodd) [reply Governance.md:179]:
```markdown
##### Maintainers

Maintainers for the repositories in the pkgjs organization are managed through teams for each repo. When a new repo is created, the user who requested it is automatically added and given management capability on the repo's team.  The initial committer or any Admin member can add/remove members after an issue is opened on the Package Maintenance repo following the normal review guidelines.  Members of the repo team will also be given publish rights on the registry or any other roles required to ship the library/package/code.
```

What do you think of something like this?  I think this will give us a bit more clarity so that we know everyone with publish rights as well, which I think right now is apparently just me, so that will also need to be fixed.

**Comment by mhdawson** [comment]
@wesleytodd did you mean the `original requester` by this in your suggestion above as opposed to  `The initial committer ` ?  With that change it looks good to me.

File: Governance.md (hunk 21 above)
Comment (mhdawson) [reply Governance.md:179]: @wesleytodd did you mean the original requester by this in your suggestion above as opposed to The initial committer ? With that change it looks good to me. Otherwise +1

File: Governance.md [hunk 29]
```diff
@@ -0,0 +1,262 @@
...
+Maintainers for the repositories in the Pkgjs repository are managed
+through the [pkgjs-maintainers](https://github.com/orgs/nodejs/teams/pkgjs-maintainers/)
+team.  Administrative members are given the maintainer role for that team
+and can add/remove members as appropriate.
+
+##### Landing PRs
+
+The package maintenance team policy on landing a PR in the repositories within the Pkgjs
+repository is for there to be:
+- At least 2 approvals from package-maintenance members
```
Comment (mhdawson) [inline Governance.md:187]:
```suggestion
- At least 1 approval from package-maintenance members 
```

File: Charter.md (hunk 14 above)
Comment (dominykas) [reply Charter.md:27]: TIL.

**Comment by mhdawson** [comment]
@wesleytodd @dominykas @ljharb I think I've addressed the comments/discussion from the last meeting. Please take a look.

File: README.md [hunk 30]
```diff
@@ -65,15 +80,17 @@ to be accepted: it is a manual workflow, so it could take some days (we are work
 
 ## Pull Request Merging Policy
 
-The package maintenance team policy on PR closure is for there to be
-- At least 4 approvals
+The package maintenance team policy on landing a PR n the nodejs/package-maintenance
```
Comment (dominykas) [inline README.md:83]:
```suggestion
The package maintenance team policy on landing a PR in the nodejs/package-maintenance
```

File: Governance.md [hunk 31]
```diff
@@ -0,0 +1,269 @@
...
+the agenda, and may be suggested as any other agenda
+item (see "WG Meetings" below).
+
+If an addition or removal is proposed, this should be raised in an 
+issue, tagged for the next agenda, and concensus should be reached
+in the issue. This is to ensure
+that all Administrative  members are given the opportunity to
+participate in all membership decisions. The addition or removal
+is considered approved once the issue has been open for 2 meetings
+or 14 days whichever is longer,and at least half the Administrative
```
Comment (dominykas) [inline Governance.md:45]:
```suggestion
or 14 days whichever is longer, and at least half the Administrative
```

File: Governance.md [hunk 32]
```diff
@@ -0,0 +1,269 @@
...
+cannot veto or remove items.
+
+The moderator is responsible for summarizing the discussion of each
+agenda item and sends it as a pull request after the meeting.
+
+### Repository Management
+All repositories under the management of the package maintenance team must include:
+
+* LICENCE.md from the list approved by the OpenJS Foundation
+* CODE_OF_CONDUCT.md referencing the Node.js Code of conduct in the admin repo.
```
Comment (dominykas) [inline Governance.md:89]: Do we need to adjust the wording to cover the fact that this will be in the `.github` repo?

File: Governance.md [hunk 33]
```diff
@@ -0,0 +1,269 @@
...
+Maintainers for the nodejs/package-maintenance repo are managed
+through the [package-maintenance](https://github.com/orgs/nodejs/teams/package-maintenance/)
+team.  Administrative members are given the maintainer role for that team
+and can add/remove members as they join or leave the team.
+
+##### Landing PRs
+
+The package maintenance team policy on landing a PR in this repository
+is for there to be:
+- At least 4 approvals from rebular members other than the author of the PR
```
Comment (dominykas) [inline Governance.md:107]:
```suggestion
- At least 4 approvals from regular members other than the author of the PR
```

File: Governance.md [hunk 34]
```diff
@@ -0,0 +1,269 @@
...
+
+In certain cases, OpenJS Cross Project Council and/or OpenJS Foundation Board
+of Directors approval may also be required. (Most likely in the case of
+transfeering a repository into or out of the organization).
+
+##### Maintainers
+
+Maintainers for the repositories in the Pkgjs repository are managed
+through a team created for each repository.  They will be named as
+[pkgjs-maintainers](https://github.com/orgs/nodejs/teams/REPO-maintainers/)
```
Comment (dominykas) [inline Governance.md:180]:
```suggestion
[REPO-maintainers](https://github.com/orgs/pkgjs/teams/REPO-maintainers/)
```

Blindly guessing what was the intent here?

File: Governance.md [hunk 35]
```diff
@@ -0,0 +1,269 @@
...
+where REPO is the name of the repository.  Administrative members are given
+the maintainer role these team and can add/remove members as appropriate.
+In addition all maintainers for a given repository can add/remove
+maintainers are given the maintainer role fot the teams for which they are
```
Comment (dominykas) [inline Governance.md:184]:
```suggestion
maintainers are given the maintainer role for the teams for which they are
```

File: Governance.md [hunk 36]
```diff
@@ -0,0 +1,269 @@
...
+- No blocking reviews
+- 72 hours since the PR was openened 
+
+#### pkgjs organization
+All members of the pkgjs organization will be required to have 2FA enabled, 
+and the repository will be configured to enforce this requirement.
+
+##### Adding or removing repositories
+
+Any repository created under the Pkgjs.js GitHub Organization is considered to be
```
Comment (dominykas) [inline Governance.md:153]:
```suggestion
Any repository created under the Pkgjs GitHub Organization is considered to be
```

File: Governance.md (hunk 32 above)
Comment (mhdawson) [reply Governance.md:89]: There was some discussion in the OpenJS Cross project council that at least for the CODE_OF_CONDUCT.md we may require a copy or link in the repo, otherwise it's not discoverable since you don't see it when looking in or cloning th erepo.

Comment (dominykas) [reply Governance.md:89]: Would a link in the README suffice?

That said, there's something comforting about that file just being present _at the root_ of the repo.

Comment (mhdawson) [reply Governance.md:89]: That was the sentiment, it's most easily discover-able and clear that there is one if the file is there at the root of the project.  I think it could be a link to the shared one so that after it's committed it never needs an update.

**Comment by mhdawson** [comment]
PR to request chartering: https://github.com/nodejs/TSC/pull/880
